src/cron/bundle.rs

```diff
@@ -38,7 +38,7 @@ pub struct TxReceipt {
     signature: String,
 }
 
-pub async fn get_bundler() -> Result<Bundler, ValidatorCronError> {
+pub fn get_bundler() -> Result<Bundler, ValidatorCronError> {
     Ok(Bundler {
         address: "OXcT1sVRSA5eGwt2k6Yuz8-3e3g9WJi5uSE99CWqsBs".to_string(),
         url: "https://node1.bundlr.network/".to_string(),
```

Comment:
Can you add `FIXME` comment here to flag that we have hardcoded values here that we should get from context.

Reply:
Actually no need to mark this with `FIXME`, I just created a PR to fix this: #25